js/qz-tray.js

```diff
@@ -1275,28 +1291,20 @@ var qz = (function() {
              * Send commands over a serial port.
              * Any responses from the device will be sent to serial callback functions.
              *
-             * @param {string} port An open port to send data over.
-             * @param {string|Object} data The object containing data and metadata to send to the serial device.
-             *  @param {string} [data.type='PLAIN'] Valid values <code>[PLAIN | FILE]</code>
-             *  @param {string} data.data The data to send to the serial device.
-             * @param {Object} [properties] DEPRECATED: Properties of data being sent over the serial port.
-             *  @param {string} [properties.baudRate=9600]
-             *  @param {string} [properties.dataBits=8]
-             *  @param {string} [properties.stopBits=1]
-             *  @param {string} [properties.parity='NONE'] Valid values <code>[NONE| EVEN | ODD | MARK | SPACE]</code>
-             *  @param {string} [properties.flowControl='NONE'] Valid values <code>[NONE | XONXOFF | XONXOFF_OUT | XONXOFF_IN | RTSCTS | RTSCTS_OUT | RTSCTS_IN]</code>
+             * @param {string} port An open serial port to send data.
+             * @param {string|Array<string>|Object} data Data to be sent to the serial device.
+             *  @param {string} [data.type='PLAIN'] Valid values <code>[FILE | PLAIN | HEX?? | BASE64??]</code>
```

Comment:
I see that `HEX` and `BASE64` options were added on documentation, but not on  `SerialUtilities#SerialDataType`. Are there any plans on supporting both of those types?